libpod/networking_common.go

```diff
@@ -632,6 +641,17 @@ func (r *Runtime) ConnectContainerToNetwork(nameOrID, netName string, netOpts ty
 	return ctr.NetworkConnect(nameOrID, netName, netOpts)
 }
 
+// hostInterfaceName takes a network name, a partial or a full network ID and
+// returns the host interface name.
+// return an empty string in case of any error.
+func (r *Runtime) hostInterfaceName(nameOrID string) (string, error) {
+	net, err := r.network.NetworkInspect(nameOrID)
+	if err != nil {
+		return ""
```

Comment:
It might be nice to have a log.debug statement here with the error listed.

Reply:
Makes sense. Should the function also specifically check for driver "bridge" & "macvlan" or is it okay to return the interface name unconditionally?